netbox/netbox/views/generic/object_views.py

```diff
@@ -334,6 +335,18 @@ def get(self, request, *args, **kwargs):
         obj = self.get_object(**kwargs)
         form = ConfirmationForm(initial=request.GET)
 
+        using = router.db_for_write(obj._meta.model)
+        collector = Collector(using=using)
+        collector.collect([obj])
+        deletion_objects = []
+        if collector.instances_with_model():
+            for model, instance in collector.instances_with_model():
+                # we could ignore the instance == obj so that the list doesnt contain itself...
+                deletion_objects.append({
+                    "modelname":f"{model.__name__}", 
```

Comment:
What's the need of the f-string here? Isn't the name already a string?

Reply:
cleaned up. was a leftover from my first pass